Resources/Prototypes/DeltaV/Catalog/Fills/Lockers/suit_storage.yml

```diff
@@ -0,0 +1,17 @@
+- type: entity
+  id: SuitStorageSecDeltaV
+  parent: SuitStorageSec
+  suffix: Security, DeltaV
```

Comment:
wont this inherit the hidespawnmenu, need categories: []

Reply:
@deltanedas 
![image](https://github.com/user-attachments/assets/c476adba-f797-4a4d-b162-1f54e6b9e4b9)

Reply:
real